Makefile

```diff
@@ -113,16 +112,33 @@ srpm: .srpm-done
 
 rpm: .rpm-done
 
-ubuntu-trusty:
-	cp packaging/ubuntu-trusty/ecs.conf ecs.conf
-	tar -czf ./amazon-ecs-init_${VERSION}.orig.tar.gz ecs-init ecs.conf scripts README.md
+ARCH:=$(shell uname -m)
+ifeq (${ARCH},x86_64)
+	AGENT_URL=https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-v${VERSION}.tar
+else ifeq (${ARCH},aarch64)
+	AGENT_URL=https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-arm64-v${VERSION}.tar
+endif
+
+/tmp/ecs-agent-version:
+	echo "${VERSION}" >/tmp/ecs-agent-version
```

Comment:
Don't use a static name in a world-writable directory for this. Consider what happens if /tmp/ecs-agent-version is a symlink to some other file.

You should either invoke `mktemp` to create a file with an unpredictable name, or create this file in the build directory somewhere.

Reply:
I've changed this to put ecs-agent.tar in the BUILDROOT directory.

ecs-agent-version, as it's not a binary file, I put it in ../BUILDROOT. I was not able to put it in BUILDROOT as dpkg did not like that it was an unmanaged/changed file.